.github/workflows/build.yml

```diff
@@ -0,0 +1,94 @@
+name: Build
+
+on:
+    push:
+        branches: [main, next, master, 2.5, 3.4, 4.3, 5.1, 6.2]
+    pull_request:
+    release:
+        types: [created]
+
+jobs:
+    tests:
+        runs-on: ubuntu-latest
+        name: Build and test
+        strategy:
+            matrix:
+                php: [7.3, 7.4]
+                composer-flags: [""]
+                composer-extras: [""]
+                stability: ["stable"]
+                include:
+                    -   php: 7.3
+                        publish-phar: true
+                    -   php: 7.3
+                        composer-flags: --prefer-lowest
+                    -   php: 7.3
+                        composer-extras: "symfony/event-dispatcher ^3.4 symfony/contracts ^2.0"
+                    -   php: 7.4
+                        stability: "dev"
+
+        env:
+            COMPOSER_ROOT_VERSION: dev-master
+
+        steps:
+            -   uses: actions/checkout@v2
+
+            -   name: Setup PHP
+                uses: shivammathur/setup-php@v2
```

Comment:
How about caching this?

Reply:
There was no caching on travis. I suggest we reproduce the same build first, and then see if caching is worth it.

Build on Github takes slightly above a minute. Caching might actually slow it down.

Reply:
Ok

Reply:
@drupol I was wrong, Travis is configured to use caching:

```
cache:
  directories:
    - $HOME/.composer/cache
```

I'm still opting to do it once this PR is merged. Perhaps you'd be up for it? ;)

Reply:
I'm ok to take care of it, but it's very easy to implement, see this comment: https://github.com/phpspec/phpspec/pull/1337#issuecomment-716707082

The file linked in there add steps to enable caching:

```
            - name: Get Composer Cache Directory
              id: composer-cache
              run: echo "::set-output name=dir::$(composer config cache-files-dir)"

            - name: Cache dependencies
              uses: actions/cache@v2
              with:
                  path: ${{ steps.composer-cache.outputs.dir }}
                  key: ${{ runner.os }}-composer-${{ hashFiles('**/composer.json') }}
                  restore-keys: ${{ runner.os }}-composer-
```

Reply:
Yeah, I know how to do it, I just don't think it's worth it! :)

Reply:
Up to you to decide, I'm just proposing :-)